File: WhereAreYou.Core/Responses/MeetResponse.cs
using System.Collections.Generic;
using WhereAreYou.Core.Entity;

namespace WhereAreYou.Core.Responses
{
    public class MeetResponse
    {
        public Meet Meet { get; set; }
        public IEnumerable<UserPosition> Users { get; set; }
        public IEnumerable<AdvertPosition> Adverts { get; set; }
        public UserPosition CurrentUser { get; set; }
        public Location CenterPoint { get; set; }
        public int ZoomLevel { get; set; }
    }
}

File: WhereAreYou.Web/ClientApp/src/app/models/meet-response.ts
import { Position } from './position';
import { Location } from './location';
import { Meet } from './meet';
import { UserPosition } from './user-position';

export class MeetResponse {
  meet: Meet;
  users: UserPosition[];
  currentUser: UserPosition;
  centerPoint: Location;
  zoomLevel: number;

  constructor() {
    this.currentUser = new UserPosition();
    this.centerPoint = new Location();
    this.users = new Array<UserPosition>(); //TODO: Change to Position
  }
}

